content/requests/post_oauth2_token.yml

```diff
@@ -135,3 +136,22 @@ properties:
     description: |-
       Full URL for the file that the token should be generated for.
     example: https://api.box.com/2.0/files/123456
+
+  box_subject_type:
+    type: string
+    example: enterprise
+    description: |-
+      Used in combination with `client_credentials` as the `grant_type`.
+    enum:
+      - enterprise
+      - user
+
+  box_subject_id:
+    type: string
+    example: '123456789'
+    description: |-
+      Used in combination with `client_credentials` as the `grant_type`.
+      Value is determined by `box_subject_type`.
+    enum:
+      - enterprise ID
+      - user ID
```

Comment:
@knmeiss So an enum is used when there is a finite list of known options for a value, for example a list of template operations https://github.com/box/box-openapi/blob/main/content/attributes/metadata_template_operation.yml#L17-L30

In this case, the value can be any enterprise ID or user ID, and although this is finite it's hardly known or static. I'd just remove these lines to be honest, and instead I'd add something to the description to explain that the subject ID can be an enterprise ID or user ID.